README.md

```diff
@@ -1,205 +1,241 @@
-# Awesome Deep Learning Project Ideas 
-[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) 
+<!-- markdownlint-disable MD033 -->
 
-**Curated List of Practical Deep Learning, Machine learning project Ideas**
- - 30+ ideas 
- - Relevant to academia and industry both
- - Range from beginner friendly to research projects
+# Awesome Deep Learning Project Ideas
 
+[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+
+A curated list of practical deep learning and machine learning project ideas
+
+- 30+ ideas
+- Relevant to both the academia and industry
+- Ranges from beginner friendly to research projects
+
+---
 
 ## Contents
 
-- [Text](#text) - including natural language under this section 
-- [Forecasting](#forecasting) - mostly Time Series and similar forecasting challenges
-- [Recommender Systems](#recommender-systems)
-- [Vision](#vision) - includes image and video processing
-- [Music and Audio](#music) - combine ideas from language, and audio to understand music
+- [Text](#text) - With some topics about Natural language processing
 
-Text
----------
-**Autonomous Tagging of Stack Overflow Questions** 
-- Make a multi-label classification system that automatically assigns tags for questions posted on a forum such as Stackoverflow or Quora 
-- Dataset: [StackLite](https://www.kaggle.com/stackoverflow/stacklite) or [10% sample](https://www.kaggle.com/stackoverflow/stacksample) 
+- [Forecasting](#forecasting) - Most of the topics in this section is about Time Series and similar forecasting challenges
 
-**Keyword/Concept Identification** 
-- Identify keywords from millions of text questions
-- Dataset: [StackOverflow Questions Sample by Facebook](https://www.kaggle.com/c/facebook-recruiting-iii-keyword-extraction/data)
+- [Recommendation Systems](#recommendation-systems)
 
-**Topics Identification**
-- Multi-label classification of printed media articles to topics
-- Dataset: [Greek Media Monitoring Multilabel Classification](https://www.kaggle.com/c/wise-2014/data)
+- [Vision](#vision) - With topics about image and video processing
+
+- [Music and Audio](#music) - These topics are bout combining ideas from language and audio to understand music
+
+- [Conclusion](#conclusion)
+
+---
+
+## Text <a name="text"></a>
```

Comment:
The internal hyperlinks work natively with Github Markdown. Why add the <a></a> tags?

Reply:
My editor automatically noticed them and changed them. Not really sure why though...

Reply:
Can we please remove those and send a new PR?

Reply:
Yes sure. Must I remove just the tags?